tests/typecheck/managers/querysets/test_from_queryset.yml

```diff
@@ -96,9 +96,9 @@
 -   case: from_queryset_returns_intersection_of_manager_and_queryset
     main: |
         from myapp.models import MyModel, NewManager
-        reveal_type(NewManager())  # N: Revealed type is "myapp.models.NewManager"
+        reveal_type(NewManager())  # N: Revealed type is "myapp.models.NewManager[<nothing>]"
         reveal_type(MyModel.objects)  # N: Revealed type is "myapp.models.NewManager[myapp.models.MyModel]"
-        reveal_type(MyModel.objects.get())  # N: Revealed type is "Any"
+        reveal_type(MyModel.objects.get())  # N: Revealed type is "myapp.models.MyModel"
```

Comment:
Interesting that we previously expected `Any` from `objects.get`..

Reply:
Yeah, it was a surprising failure...